### PR TITLE
Added info about what fulltext features are availble in which search engines

### PR DESCRIPTION
**Pull request by vidarl**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      | 3.3, 4.5
| Edition       | Content/Headless

FYI : Added feature request for supporting phrase search on elastic : https://ibexa.productboard.com/all-notes/notes/35901035

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Comment by vidarl** [comment]
I guess failed vale-check is not my fault...

**Review by MagdalenaZuba — APPROVED** [review]

**Comment by MagdalenaZuba** [comment]
No worries, just to double-check, merge is intended only for v3.3?

**Comment by vidarl** [comment]
> No worries, just to double-check, merge is intended only for v3.3?
No, sorry. 3.3 and up ( I have updated description)

**Comment by vidarl** [comment]
Don't merge this one yet.. I think I will do additional changes

**Comment by adamwojs** [comment]
Maybe we can expand "Supported syntax" section and add information on each capability which search engine support it ?

**Comment by vidarl** [comment]
> Maybe we can expand "Supported syntax" section and add information on each capability which search engine support it ?

Yes, I was planning to do exactly that... However, stuff came up so I have been delayed.. Will try to this tomorrow

**Comment by adamwojs** [comment]
@vidarl Are you still willing to continue work on this PR ?

**Comment by vidarl** [comment]
@adamwojs : Sorry for the delay in the matter. Yes, I can continue the work.
ATM I have lost write access to this repo but that is about to be fixed soon (it is in progress).

**Comment by vidarl** [comment]
@adamwojs : Done now, ready for new review

File: docs/guide/search/criteria_reference/fulltext_criterion.md
Comment (MagdalenaZuba) [inline docs/guide/search/criteria_reference/fulltext_criterion.md:22]:
```suggestion
\* When using the Elasticsearch search engine, a full text query performs an OR query by default, while the OR and AND operators return unexpected results.

\*\* When using the Legacy search engine, a full text query performs an OR query.

\*\*\* Asteriks may only be located at the beginning or end of a query.
```

**Review by MagdalenaZuba — APPROVED** [review]

**Pull request merged**
